crates/holochain/src/core/queue_consumer/countersigning_consumer.rs

```diff
@@ -1,43 +1,48 @@
 //! The queue consumer for the countersigning workflow.
 
 use super::*;
+#[cfg(feature = "unstable-countersigning")]
 use crate::core::workflow::countersigning_workflow::countersigning_workflow;
 use tracing::*;
 
 /// Spawn the QueueConsumer for the countersigning workflow
 #[instrument(skip_all)]
 pub(crate) fn spawn_countersigning_consumer(
-    space: Space,
-    workspace: Arc<CountersigningWorkspace>,
+    _space: Space,
+    _workspace: Arc<CountersigningWorkspace>,
```

Comment:
I'd prefer an `#[allow(unused)]` to renaming the parameters, because it'll be easier to remove when we stablize the feature.

Reply:
updated

Reply:
@jost-s - I see you fixed this one, but a bunch of other `_` variables got added in other places since my last review. I feel the same about them.

Reply:
That's an unjustified allegation!!! I didn't add anything while you weren't looking! You just happened to spot them now. =D

I've gone through all occurrences of the feature and fixed two other places.